src/Dotnet.Script.Tests/TestFixtures/Issue226/Issue226.csx

```diff
@@ -0,0 +1,12 @@
+#! "netcoreapp2.0"
+#r "nuget: Castle.Core,*"
+
+Process.Start(new ProcessStartInfo {
```

Comment:
Is it referencing `Castle.Core` that causes the error is it starting the process? And what is the relation here? 😄

Reply:
The error happens only when I reference one of "those" nugets, that probably reference ether new or older `System.ComponentModel.Primitives` package, that would be my guess.

Reply:
I know there were some kind of a bug when you upgrade I think from dotnet 2.0.* to 2.1.* that might be my problem. I already reinstalled it on my machine quite a few times, but I think there was another step that I needed todo, however can't find the article for this -.-.

Reply:
Does this mean that it still fails for you on Linux?

Reply:
Yes.

Reply:
Are you running .Net Core 2.1 Preview?

Reply:
```
me@my-pc ~ $ apt list --installed | grep dotnet*

WARNING: apt does not have a stable CLI interface. Use with caution in scripts.

dotnet-host/xenial,now 2.1.0-preview1-26216-03-1 amd64 [installed,automatic]
dotnet-hostfxr-2.0.5/xenial,now 2.0.5-1 amd64 [installed,automatic]
dotnet-runtime-2.0.5/xenial,now 2.0.5-1 amd64 [installed,automatic]
dotnet-runtime-deps-2.1.0-preview1-26216-03/xenial,now 2.1.0-preview1-26216-03-1 amd64 [installed,automatic]
dotnet-sdk-2.1.4/xenial,now 2.1.4-1 amd64 [installed]
```
I'm pretty sure I do :)

Reply:
I see. The Travis build uses .net Core 2.0 so that is probably why it does not fail there. 
We haven't really started to look at .Net Core 2.1 yet, but I guess it would be possible to add that to the build matrix as well. It does not make all that much sense to merge this without seeing it fail. Only then would we know where to fix it 👍

Reply:
Hmm, haven't thought that 2.1 might be a problem, I might try downgrading to 2.0 :). It's not a big deal, I made this PR, so it would be easier to see where the problem is, but since there is no problem we can close this PR.